lib/db/project.ts

```diff
@@ -0,0 +1,38 @@
+import prisma from "./prisma";
+import { Project } from "@prisma/client";
+
+/**
+ * Creates a new project.
+ *
+ * @export
+ * @async
+ * @param {Project} params Information about the project to create.
+ * @returns {Promise<Project>} The created project object.
+ */
+export async function createProject(
+  params: Partial<Project>
+): Promise<Project> {
+  const project = await prisma.project.create({
+    data: {
+      ...params,
+    },
+  });
+  return project;
```

Comment:
Nice touch returning the created / modified / deleted record. Will be useful for validation.